src/components/atoms/SfArrow/SfArrow.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../css/variables';
+
+// $component__block-property: value;
+
+.sf-arrow {
+  position: relative;
+  display: flex;
+  background-color: transparent;
+  border: 0;
+  padding: 14px 10px 14px;
+  background-color: #F1F2F3;
+  cursor: pointer;
+  transition: padding 150ms ease-in-out;
```

Comment:
What about transition for colors too?
```scss
transition: padding 150ms ease-in-out, background-color 150ms linear, color 150ms linear;
```

Reply:
![arrow2_long](https://user-images.githubusercontent.com/12138170/58545748-2ee8ce80-8204-11e9-80a1-24aa91a9fb83.gif)
![arrow2](https://user-images.githubusercontent.com/12138170/58545749-2ee8ce80-8204-11e9-82e3-a919e50c6245.gif)

I think, now looks better